webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7361,20 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.checkBotDetail = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + CURRENT_GROUP_DATA.conversationAssignee,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                                CURRENT_GROUP_DATA.CHAR_CHECK = data.data[0] ? ((data.data[0].aiPlatform == 'dialogflow' && !((data.data[0]).autoHumanHandoff)) ? true : false) : false ;
+                            },
+                    error: function () {
```

Comment:
Pls verify if it is required on data.data or data.data[0]

Reply:
data.data is returned as an empty array, not null. so will have to check the first array element